server/elm/src/Backend/Scoreboard/Model.elm

```diff
@@ -102,7 +103,46 @@ type alias UniversalInterventionData =
 
 emptyUniversalInterventionData : UniversalInterventionData
 emptyUniversalInterventionData =
-    UniversalInterventionData [] [] [] [] []
+    UniversalInterventionData Dict.empty [] [] [] []
+
+
+type alias VaccinationProgressDict =
+    Dict VaccineType (Dict VaccineDose NominalDate)
+
+
+type alias RawVaccinationData =
+    { bcg : EverySet NominalDate
+    , opv : EverySet NominalDate
+    , dtp : EverySet NominalDate
+    , pcv13 : EverySet NominalDate
+    , rotarix : EverySet NominalDate
+    , ipv : EverySet NominalDate
+    , mr : EverySet NominalDate
+    }
+
+
+type alias VaccinationValue =
+    { administeredDoses : EverySet VaccineDose
+    , administrationDates : EverySet NominalDate
+    }
+
+
+type VaccineType
+    = VaccineBCG
+    | VaccineOPV
+    | VaccineDTP
```

Comment:
The list of vaccines seems to be a thing that will change in the future. Wouldn't it be better to make it editable?
The list of required vaccines typically change in every country from time to time.

Reply:
@AronNovak Every vaccine got it's own adminiustration logic.
Number of doses, period between doses, and others, which are even more complex.
So, this whole logic is too complex to be configurable. Not unpossible, but not worth the effort.

We may revise in future.